x-pack/plugins/security_solution/public/entity_analytics/api/api.ts

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+import type { SnakeToCamelCase } from '@kbn/cases-plugin/common/types';
```

Comment:
We should not import helpers from other plugins

Reply:
So what happens here then?

I actually created my own utility first, then realised it already existed elsewhere. Does that mean I would add a duplicate just for security?

Reply:
Yes, inside Security, or in `packages/kbn-utility-types`, so it will be available across kibana.

Reply:
Will duplicate for this PR within Entity Analytics. Will open up a follow up PR to extract to a more reusable place